authz/sdk_end2end_test.go

```diff
@@ -420,31 +420,29 @@ func (s) TestSDKFileWatcher_ValidPolicyRefresh(t *testing.T) {
 	if err := os.WriteFile(file, []byte(valid2.authzPolicy), os.ModePerm); err != nil {
 		t.Fatalf("os.WriteFile(%q) failed: %v", file, err)
 	}
-	// Wait 30 ms for background go routine to read the updated files.
-	time.Sleep(30 * time.Millisecond)
 
 	// Verifying authorization decision.
-	_, err = client.UnaryCall(ctx, &pb.SimpleRequest{})
 	numRetries := 0
 	reloadSuccess := false
-	got := status.Convert(err)
+	var gotStatus *status.Status
 	for numRetries <= 20 {
-		if got.Code() == valid2.wantStatusCode && got.Message() == valid2.wantErr {
+		_, err = client.UnaryCall(ctx, &pb.SimpleRequest{})
+		if gotStatus = status.Convert(err); gotStatus.Code() == valid2.wantStatusCode && gotStatus.Message() == valid2.wantErr {
 			reloadSuccess = true
 			break
 		}
-		time.Sleep(10 * time.Millisecond)
+		time.Sleep(100 * time.Millisecond)
```

Comment:
10-50ms seems slightly better here -- given the refresh interval is 100ms, we might _just_ miss a reload and be stuck here for an extra 100ms unnecessarily.

Note that, while 100ms doesn't seem like much, we sometimes have flaky tests that need to be run thousands of times to reproduce, so minimizing even small delays is always worthwhile.

Reply:
Sure thing. Using 20 ms as the retry interval.